firestore/transaction.go

```diff
@@ -136,11 +138,14 @@ func (c *Client) RunTransaction(ctx context.Context, f func(context.Context, *Tr
 			// Prefer f's returned error to rollback error.
 			return err
 		}
+		t.ctx = trace.StartSpan(t.ctx, "cloud.google.com/go/firestore.Client.Commit")
```

Comment:
Is this the correct lable? Seems like this may be tracing (externally) a gapic level call?

Also, I think this exact label is also used at https://github.com/googleapis/google-cloud-go/pull/2942/files#diff-f5e901925d383c72807f3f3336f0d2681b57c12896e1c8a89051191d19c207e9R296. 

This seems non-ideal as it will overload the label?

Reply:
I think this path matches the t.c.c.Commit call. Is the other file generated so we have to do this here?

Reply:
Right, the other file is generated so yes.